scripts/core/editor3/html/from-html/index.ts

```diff
@@ -159,21 +163,18 @@ class HTMLParser {
 
     /**
      * Convert link entity data to editor3 format
-     *
-     * @param {ContentState} initialState
-     * @returns {ContentState}
      */
-    processLinks(initialState) {
+    processLinks(initialState: ContentState): ContentState {
         let contentState = initialState;
 
         contentState.getBlocksAsArray().forEach((block) => {
-            block.findEntityRanges((characterMetadata) => characterMetadata.getEntity(), (start, end) => {
+            block.findEntityRanges((characterMetadata) => Boolean(characterMetadata.getEntity()), (start, _end) => {
```

Comment:
Is this `Boolean(characterMetadata.getEntity())` different than `characterMetadata.getEntity() != null`?

Reply:
I hope not

Reply:
If what you intend to check is whether the value is null/undefined I suggest using `!= null` since it's simple and doesn't require to read a spec to understand all possible outcomes.

Reply:
Yeah I didn't give it much thought, this was not part of the task, it was for tsc not complaining about it, but it was like this already

Reply:
Can you update it to `characterMetadata.getEntity() != null`?